src/pudl/metadata/resources/ferc1.py

```diff
@@ -837,7 +837,7 @@
         "field_namespace": "ferc1",
         "etl_group": "outputs",
         "sources": ["ferc1"],
-        "create_database_schema": False,
+        "create_database_schema": True,
```

Comment:
I think you're right, but I also kind of like having it included explicitly, since it makes it clear that there's a choice to be made on each resource definition.

Reply:
Agreed except that if this was the case we'd want this to be included in all the other assets as well. I'm going to remove this for now to preserve consistency across assets.